src/scenes/ProgressReports/Detail/WorkflowEventList.tsx

```diff
@@ -0,0 +1,71 @@
+import { ArrowForward } from '@mui/icons-material';
+import { Box, Card, Collapse, Divider, Stack, Typography } from '@mui/material';
+import { ProgressReportStatusLabels as StatusLabels } from '~/api/schema/enumLists';
+import { FormattedDate } from '~/components/Formatters';
+import { RichTextView } from '~/components/RichText';
+import { WorkflowEventFragment } from './WorkflowEvent.graphql';
+
+interface WorkFlowEventListProps {
+  events?: readonly WorkflowEventFragment[];
+  showNotes?: boolean;
+}
+
+export const WorkFlowEventList = ({
+  showNotes,
+  events,
+}: WorkFlowEventListProps) => {
+  if (!events) {
+    return null;
+  }
+
+  const statusEvents = [
+    {
+      ...events[0]!,
+      status: 'NotStarted' as keyof typeof StatusLabels,
+    },
+    ...events,
+  ];
+
+  return (
+    <Card sx={{ p: 4, overflowY: 'auto' }} elevation={0}>
+      <Stack
+        component="ul"
```

Comment:
Again here, I was just following the design.  So do we want the numbers to show or do we want to still remove the numbers with 'list-Style: none"?

Reply:
Yeah. This is an ordered list so we should use <ol>. The fact that we don't want to style it that way is a css concern. Changing the list style to none makes sense.

It could make sense to use MUI's List components. But don't feel like you need too. May be more work than worth it.

Reply:
Got it.  I will take a look at the MUI list component and see how I feel about it.